src/config.ts

```diff
@@ -8,5 +8,6 @@ export let config: IConfig = {
   timeToYield: 50,
   instrument: false,
   uploadHandler: null,
-  debug: false
+  debug: false,
+  plugins: ["viewport", "layout", "pointer", "performance"]
```

Comment:
In theory, no. In reality we are fine right now but we should add unit tests to ensure we do not unintentionally pick up unexpected dependencies that force us to stick to a certain order.